kokoro/ubuntu/periodic.sh

```diff
@@ -13,5 +13,22 @@ mvn -B clean install
 # https://stackoverflow.com/questions/3459928/running-a-specific-maven-plugin-goal-from-the-command-line-in-a-sub-module-of-a/26448447#26448447
 # https://stackoverflow.com/questions/11091311/maven-execjava-goal-on-a-multi-module-project
 cd dashboard
+
+# Step 1: Generate dashboards for the released BOMs with the latest DashboardMain class
+# For example https://storage.googleapis.com/cloud-opensource-java-dashboard/dashboard/target/1.1.0/dashboard.html
+
+# Querying https://search.maven.org/classic/#api with
+# groupId:com.google.cloud AND artifactId:libraries-bom, receiving latest (sorted) 10 versions.
+SONATYPE_RESPONSE=`curl 'https://search.maven.org/solrsearch/select?q=g:%22com.google.cloud%22+AND+a:%22libraries-bom%22&core=gav&rows=10&wt=json'`
+# Example: '1.0.0 1.1.0'
+VERSIONS=`echo $SONATYPE_RESPONSE | perl -nle 'print $1 while m/"v":"(.+?)"/g'`
+for VERSION in $VERSIONS; do
+  mvn -B exec:java -Dexec.mainClass="com.google.cloud.tools.opensource.dashboard.DashboardMain" \
+    -Dexec.arguments="-c com.google.cloud:libraries-bom:${VERSION}"
+  mv target/dashboard target/${VERSION}
+done
+
+# Step 2: Generate dashboard for the snapshot
+# For example https://storage.googleapis.com/cloud-opensource-java-dashboard/dashboard/target/dashboard/dashboard.html
```

Comment:
`dashboard/target/dashboard` seems giving no value. With Kokoro job config (see above) and one `mv` command here, the URL for the snapshot BOM can become 

```
https://storage.googleapis.com/cloud-opensource-java-dashboard/snapshot/dashboard.html
```